repos/system_upgrade/common/actors/removeupgradefailedflag/libraries/removeupgradefailedflag.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from leapp.libraries.stdlib import api
+
+LEAPP_UPGRADE_FAILED_FLAG_FILE = '/root/tmp_leapp_py3/.leapp_upgrade_failed'
```

Comment:
remove the whole directory if it exists as it should not be present there anyway. And rename the actor to something like: `removeprevupgradeartifacts`, `removepythonwoarkaounddir`,  .... keeping on you